Filters/rules.txt

```diff
@@ -573,6 +573,12 @@
 !
 ! Special block section, requires explanation for the rule
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/160610
+199.167.138.162
```

Comment:
As I remember, by DNS filtering can be blocked only domain, not IPs.

Reply:
You saw Slack discussion, right? At least working on adguard DNS. Will check on apps.

Reply:
Working on AG for Android too.

Reply:
Working, e.g. latest tweet `https://twitter.com/NaomiSuzuki_/status/1699788108894728501`. They're revolving. And twitter search `199.167.138.162` `199.167.138.163` indicates the IP has been stably used.

Reply:
It is dangerous to block fresh IPs used for phishing. Hoster may ban them and sell ip to legal client.